components/apimgt/org.wso2.carbon.apimgt.keymgt/src/test/resources/dbscripts/h2.sql

```diff
@@ -1033,7 +1033,7 @@ CREATE TABLE IF NOT EXISTS AM_API_COMMENTS (
 );
 
 CREATE TABLE IF NOT EXISTS AM_API_RATINGS(
-    RATING_ID INTEGER AUTO_INCREMENT,
+    RATING_ID VARCHAR(255) NOT NULL,
```

Comment:
This will require migration. Shall we track this in the migration github issue?

Reply:
Found that you have already added. Thanks.
https://github.com/wso2/product-apim/issues/5464